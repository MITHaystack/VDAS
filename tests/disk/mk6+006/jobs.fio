[disk0]
rw=write
iodepth=8
directory=/mnt/disk0
write_bw_log=disk0_write
fill_device=1

[disk1]
rw=write
iodepth=8
directory=/mnt/disk1
write_bw_log=disk1_write
fill_device=1

[disk2]
rw=write
iodepth=8
directory=/mnt/disk2
write_bw_log=disk2_write
fill_device=1

[disk3]
rw=write
iodepth=8
directory=/mnt/disk3
write_bw_log=disk3_write
fill_device=1

[disk4]
rw=write
iodepth=8
directory=/mnt/disk4
write_bw_log=disk4_write
fill_device=1
stonewall

[disk5]
rw=write
iodepth=8
directory=/mnt/disk5
write_bw_log=disk5_write
fill_device=1

[disk6]
rw=write
iodepth=8
directory=/mnt/disk6
write_bw_log=disk6_write
fill_device=1

[disk7]
rw=write
iodepth=8
directory=/mnt/disk7
write_bw_log=disk7_write
fill_device=1
