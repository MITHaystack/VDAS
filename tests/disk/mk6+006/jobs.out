disk0;0;0;0;0;0;0;0;0.000000;0.000000;0;0;0.000000;0.000000;0;0;0.000000%;0.000000;0.000000;958541136;100787;9738784;0;0;0.000000;0.000000;2;1924806;39.844427;1478.916676;13723;1172960;25.097861%;99027.375385;28835.395078;2.854576%;11.450065%;768929;0;503;100.0%;0.0%;0.0%;0.0%;0.0%;0.0%;0.0%;0.00%;25.11%;72.69%;0.64%;0.91%;0.53%;0.03%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.04%;0.03%;0.01%;0.01%;0.01%;0.00%;0.01%;0.00%

disk1;0;0;0;0;0;0;0;0.000000;0.000000;0;0;0.000000;0.000000;0;0;0.000000%;0.000000;0.000000;961349976;103973;9468053;0;0;0.000000;0.000000;2;5325753;38.597638;1435.108861;22205;1071792;25.871386%;102079.432522;23198.031724;2.963197%;11.753220%;760243;0;494;100.0%;0.0%;0.0%;0.0%;0.0%;0.0%;0.0%;0.00%;24.89%;73.04%;0.53%;0.90%;0.53%;0.03%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.04%;0.03%;0.01%;0.01%;0.00%;0.00%;0.00%;0.01%

disk2;0;0;0;0;0;0;0;0.000000;0.000000;0;0;0.000000;0.000000;0;0;0.000000%;0.000000;0.000000;961351096;102980;9559356;0;0;0.000000;0.000000;2;4392393;38.974445;1446.609472;3;1075600;25.634928%;101146.454120;24665.704036;2.944644%;11.768346%;765568;0;497;100.0%;0.0%;0.0%;0.0%;0.0%;0.0%;0.0%;0.00%;24.20%;73.58%;0.66%;0.91%;0.53%;0.03%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.04%;0.03%;0.01%;0.01%;0.00%;0.00%;0.01%;0.01%

disk3;0;0;0;0;0;0;0;0.000000;0.000000;0;0;0.000000;0.000000;0;0;0.000000%;0.000000;0.000000;961349580;104660;9405875;0;0;0.000000;0.000000;2;5368255;38.340370;1427.061230;24270;1081296;26.041896%;102752.208631;24111.844462;2.980191%;11.734540%;758785;0;492;100.0%;0.0%;0.0%;0.0%;0.0%;0.0%;0.0%;0.00%;25.33%;72.69%;0.44%;0.90%;0.53%;0.03%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.04%;0.03%;0.01%;0.01%;0.01%;0.00%;0.00%;0.01%

disk4;1;0;0;0;0;0;0;0.000000;0.000000;0;0;0.000000;0.000000;0;0;0.000000%;0.000000;0.000000;961351692;100147;9829707;0;0;0.000000;0.000000;2;2112905;40.088749;1444.886084;15;192242;25.483659%;98286.395644;21286.611819;2.888357%;11.485328%;785505;0;504;100.0%;0.0%;0.0%;0.0%;0.0%;0.0%;0.0%;0.00%;23.06%;74.82%;0.56%;0.84%;0.60%;0.03%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.04%;0.03%;0.01%;0.01%;0.00%;0.00%;0.01%;0.01%

disk5;1;0;0;0;0;0;0;0.000000;0.000000;0;0;0.000000;0.000000;0;0;0.000000%;0.000000;0.000000;958542152;98518;9963049;0;0;0.000000;0.000000;2;1693116;40.766132;1483.206051;897;722936;25.073528%;96704.587032;22228.385696;2.840667%;11.494424%;791005;0;510;100.0%;0.0%;0.0%;0.0%;0.0%;0.0%;0.0%;0.00%;22.03%;75.75%;0.65%;0.82%;0.62%;0.03%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.03%;0.03%;0.01%;0.01%;0.01%;0.00%;0.01%;0.00%

disk6;1;0;0;0;0;0;0;0.000000;0.000000;0;0;0.000000;0.000000;0;0;0.000000%;0.000000;0.000000;961349588;103564;9505384;0;0;0.000000;0.000000;2;523148;38.743494;1397.642876;19857;214299;26.357879%;101658.120199;22760.418620;2.971464%;11.959865%;774641;0;494;100.0%;0.0%;0.0%;0.0%;0.0%;0.0%;0.0%;0.00%;22.60%;75.33%;0.52%;0.84%;0.60%;0.03%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.04%;0.03%;0.01%;0.01%;0.01%;0.00%;0.00%;0.00%

disk7;1;0;0;0;0;0;0;0.000000;0.000000;0;0;0.000000;0.000000;0;0;0.000000%;0.000000;0.000000;961350456;101206;9726895;0;0;0.000000;0.000000;2;399611;39.665515;1424.139284;17815;182368;25.749027%;99309.876007;21495.584182;2.913664%;11.713122%;779675;0;501;100.0%;0.0%;0.0%;0.0%;0.0%;0.0%;0.0%;0.00%;22.39%;75.39%;0.67%;0.84%;0.60%;0.03%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.04%;0.03%;0.01%;0.01%;0.00%;0.00%;0.00%;0.00%

